index.ts

```diff
@@ -18,7 +18,7 @@
  * @module ibm-cloud-sdk-core
  */
 
-export { BaseService } from './lib/base_service';
+export { BaseService, UserOptions } from './lib/base_service';
```

Comment:
If you are going to export `UserOptions`, should you not also export `HeaderOptions`?